packages/core/src/http-middlewares/after/throw-for-stale-auth.js

```diff
@@ -1,17 +0,0 @@
-'use strict';
-
-const errors = require('../../errors');
-
-/**
- * The server-side will raise `RefreshAuthError` when `autoRefresh === true`.
- * Once we always run `throwForStatus` or a custom `afterResponse`, we can drop `throwForStaleAuth`.
- */
-const throwForStaleAuth = resp => {
```

Comment:
No more need for this as `throwForStatus` will throw for 401 and on the server-side we check if that should result in a refresh for the app's definition.